control-plane/grpc/src/operations/volume/traits_snapshots.rs

```diff
@@ -270,13 +270,20 @@ impl VolumeSnapshotState {
 pub struct VolumeSnapshots {
     /// Snapshot entries.
     pub entries: Vec<VolumeSnapshot>,
+    /// Next token for the paginated response.
+    pub next_token: Option<u64>,
```

Comment:
Does it need to be pub if we have a getter?

Reply:
It's currently required by core/volume/service to instantiate in grpc response.